p1_mqtt/cli.py

```diff
@@ -161,6 +176,11 @@ def p1_mqtt() -> None:
         help="Send timestamps to MQTT in milliseconds instead of seconds. "
         "This will only affect p1mqtt* timestamp values",
     )
+    parser.add_argument(
+        "--mqtt-rate",
+        action="store_true",
```

Comment:
```suggestion
        type=int,
```

We also might want to check later that the final value of `config["mqtt_rate"]` is a positive integer, or 0.